test/BinaryToText.cpp

```diff
@@ -285,6 +288,8 @@ INSTANTIATE_TEST_CASE_P(LoopControlMasks, RoundTripInstructionsTest,
                             "OpLoopMerge %1 %2 Unroll\n",
                             "OpLoopMerge %1 %2 DontUnroll\n",
                             "OpLoopMerge %1 %2 Unroll|DontUnroll\n",
+                            "OpLoopMerge %1 %2 DependencyInfinite\n",
+                            "OpLoopMerge %1 %2 DependencyLength 8\n",
```

Comment:
Most of the existing tests test the assembler parts in TextToBinary.*  and the diassembler/round trip is in BinaryToText.  I'd rather not change the pattern.  Yes, there are some tests in various TextToBinary.* that are round-trip, but that's the exception, not the rule.